modules/ocf_desktop/manifests/udev.pp

```diff
@@ -0,0 +1,8 @@
+class ocf_desktop::udev {
+  package { ['libu2f-host']:; }
```

Comment:
This looks like it should be `libu2f-host0` since `libu2f-host` isn't in the package archives.